### PR TITLE
Update Paella 7 default theme

### PR DESCRIPTION
**Pull request by miesgre**
This is the result of the work done with ETHZ in design and accessibility. 

### Your pull request should…

* [x] have a concise title
* [ ] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [ ] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: modules/engage-paella-player-7/src/icons/forward-icon.svg
Comment (mliradelc) [inline modules/engage-paella-player-7/src/icons/forward-icon.svg:1]: You have the backward icon without number, why this SVG uses number (or the number is by default?)?

**Review by mliradelc — CHANGES_REQUESTED** [review]
Hi Miguel, thanks for the PR, the new design is awesome and is much moderner than the original. Also, the colors now are in harmony with the rest of the UI.

Now we have found a little visual bug:

Screenshot 2023-05-08 at 08 50 22

The forward and backward icons are not aligned with the other buttons.

Screenshot 2023-05-08 at 09 02 20

When you highlight the buttons, the difference is bigger, making an unpleasant to sight.

File: modules/engage-paella-player-7/src/icons/forward-icon.svg
Comment (miesgre) [reply modules/engage-paella-player-7/src/icons/forward-icon.svg:1]: The text is not important. It is replaced in paella plugins by the time configured in the plugins. The text in the svg is only used in the editor to check the position.

**Comment by miesgre** [comment]
> Hi Miguel, thanks for the PR, the new design is awesome and is much moderner than the original. Also, the colors now are in harmony with the rest of the UI.
> 
> Now we have found a little visual bug:
> 
> Screenshot 2023-05-08 at 08 50 22
> 
> The forward and backward icons are not aligned with the other buttons.
> 
> Screenshot 2023-05-08 at 09 02 20
> 
> When you highlight the buttons, the difference is bigger, making an unpleasant to sight.

It's not a bug, it is done intentionally. ETHZ wanted it like this.
Check this: https://github.com/polimediaupv/paella-ethz/issues/13#issuecomment-1445463260

This is something that can be changed in CSS. Just change the `margin-top` property of these plugins:

Add this to `etc/ui-config/mh_default_org/paella7/custom_theme.css`
```css
.playback-bar .button-plugins .button-plugin-container button[name="es.upv.paella.backwardButtonPlugin"] div{
    margin-top: 0;
}

.playback-bar .button-plugins .button-plugin-container button[name="es.upv.paella.forwardButtonPlugin"] div{
    margin-top: 0;
}
```

**Comment by mliradelc** [comment]
In that case I think is ok.

Have you tried with a touch screen?

**Comment by miesgre** [comment]
> In that case I think is ok.
> 
> Have you tried with a touch screen?

We tested in mobile and iPad

**Comment by mliradelc** [comment]
> > In that case I think is ok.
> > Have you tried with a touch screen?
> 
> We tested in mobile and iPad

Wonderful! In that case for me is approved.

**Review by mliradelc — APPROVED** [review]
Works as intended, and the design questions were answered.

Wait for the dev meeting, if there are no objection. Should me merged before r/14.x

Pinging @wsmirnow and @stefanosgeo to take into account this PR before the branch cut

**Comment by oas777** [comment]
Thanks Max for your review - and the comment about the horizontal alignment. As Miguel indicated, we looked at the other option which was not ideal either. Also, I should mention that there will be a "final design review" of Tobira and Paella, so maybe we follow your suggestion after all.

**Comment by mliradelc** [comment]
Hi Olaf, thank you for your answer. Marcello here at cologne has suggested me this media icon's collection https://github.com/vidstack/media-icons the best, it has MIT license.

Maybe the rollback (forward) design can improve the alignment. Nevertheless, I think right now is a good job and a step forward to a better player UI.

**Comment by oas777** [comment]
Thanks for the pointer, but we already decided on the icons 
- https://github.com/feathericons/feather#feather and
- https://v1.heroicons.com/,

both with a MIT licence. And given we aligned them for Paella and Tobira, it's not easy to simply switch...

**Comment by mliradelc** [comment]
No worries, now with the new Skins options in paella, if any wants other type of icons can change it. 😄

**Pull request merged**